### PR TITLE
Update recipe for v2.2.4

### PR DESCRIPTION
**Pull request by adrianeboyd**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
@conda-forge-admin, please rerender

**Comment by adrianeboyd** [comment]
@conda-forge-admin, please restart ci

**Comment by rmax** [comment]
Confusing error https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=159833&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=6d4b912b-175d-51da-0fd9-4d30fe1eb4e7&l=452

There are win packages for srsly 1.0.2 https://anaconda.org/conda-forge/srsly/files

**Comment by adrianeboyd** [comment]
Maybe I just didn't wait long enough for the changes to propagate? Let me try again...

**Comment by adrianeboyd** [comment]
@conda-forge-admin, please restart ci

**Comment by adrianeboyd** [comment]
@conda-forge-admin, please restart ci
